Community/Tdarr_Plugin_Soggys_NVENC_HEVC_CQV_Optimised_Bitrate.js

```diff
@@ -0,0 +1,209 @@
+const details = () => ({
+  id: 'Tdarr_Plugin_Soggys_NVENC_HEVC_CQV_Optimised_Bitrate',
+  Stage: 'Pre-processing',
+  Name: 'Soggys NVENC HEVC CQ:V Optimised Bitrate',
+  Type: 'Video',
+  Operation: 'Transcode',
+  Description: `[Contains built-in filter] This plugin used nvenc and transcodes based on specificed CQ:V value.
+     Will transcode if bitrate is greater than "optimized bitrate". Optimal bitrate accounts for fps and resolution.
+     Optimized bitrate can be configured using targetCodecCompression. Smaller values target lower bitrates.
+     FFmpeg Preset can be configured, uses slow by default.
+     Low res files (720p or below) will not be transcoded
+     If files are not in hevc they will be transcoded.
+     The output container is mkv.
+     Note that you will get an "infinite transcode loop" error if target codec compression is too low compared to CQ:V
+     or CQ:V is too low compared to compression.
+     Basically: If you increment target codec compression you can lower cqv for higher quality and vice versa.
+     Thanks to JB and vdka for their plugins, this uses code from their work.\n\n`,
+  Version: '1.00',
+  Tags: 'pre-processing,ffmpeg,video only,nvenc h265,configurable',
+
+  Inputs: [
+    {
+      name: 'targetCodecCompression',
+      type: 'number',
+      defaultValue: 0.12,
+      inputUI: {
+        type: 'text',
+      },
+      tooltip: `This effects the target bitrate by assuming a compression ratio.
+
+      \\nExample:\\n
+      0.08`,
+    },
+    {
+      name: 'cqv',
+      type: 'number',
+      defaultValue: 28,
+      inputUI: {
+        type: 'text',
+      },
+      tooltip: `Enter the CQ:V value, lower is higher quality.
+
+      \\nExample:\\n
+      28`,
+    },
+    {
+      name: 'bframe',
+      type: 'number',
+      defaultValue: 0,
+      inputUI: {
+        type: 'text',
+      },
+      tooltip: `Specify amount of b-frames to use, 0-5. Use 0 to disable. (GPU must support this)
+
+      \\nExample:\\n
+      3`,
+    },
+    {
+      name: 'ten_bit',
+      type: 'string',
+      defaultValue: 'true',
+      inputUI: {
+        type: 'text',
+      },
```

Comment:
There's a type `boolean`, you can use the inputUI type `dropdown` with an options array with `'true'` and `'false'`. https://github.com/HaveAGitGat/Tdarr_Plugins/blob/1f6512e6b652d53db602f3e95738e69fc2a0b44d/examples/Tdarr_Plugin_pre1_Pre_Proc_Example.js#L33-L39